apps/core/test/services/scaffolds_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Core.Services.ScaffoldsTest do
+  use Core.SchemaCase, async: true
+  alias Core.Services.Scaffolds
+
+  describe "#tf providers" do
```

Comment:
Could we add a test that validates that all providers are implemented, maybe just doing:

```elixir
Scaffolds.available_providers()
|> Enum.map(&Scaffolds.provider/1)
```

and ensuring nothing throws?